### PR TITLE
doc update

### PR DESCRIPTION
**Pull request by tbarnes-us**
Update to sample intro that Mark requested.

### CODE REVIEW

File: docs-source/content/samples/simple/domains/model-in-image/_index.md [hunk 1]
```diff
@@ -23,15 +23,21 @@ description: "Sample for supplying a WebLogic Deploy Tool (WDT) model that the o
 
 ### Introduction
 
-This sample demonstrates:
+This sample demonstrates deploying a Model in Image domain home source type. Unlike the Domain in PV and Domain in Image domain source types, Model in Image eliminates the need to pre-create your WebLogic domain home prior to deploying your domain resource. Instead, a Model in Image deployment uses a WebLogic Deploy Tool (WDT) model to specify your WebLogic configuration.
```
Comment (rosemarymarano) [inline docs-source/content/samples/simple/domains/model-in-image/_index.md]: Unlike the Domain in PV and Domain in Image domain source types, -> Unlike Domain in PV and Domain in Image,

File: docs-source/content/samples/simple/domains/model-in-image/_index.md [hunk 2]
```diff
@@ -23,15 +23,21 @@ description: "Sample for supplying a WebLogic Deploy Tool (WDT) model that the o
 
 ### Introduction
 
-This sample demonstrates:
+This sample demonstrates deploying a Model in Image domain home source type. Unlike the Domain in PV and Domain in Image domain source types, Model in Image eliminates the need to pre-create your WebLogic domain home prior to deploying your domain resource. Instead, a Model in Image deployment uses a WebLogic Deploy Tool (WDT) model to specify your WebLogic configuration. 
+
+WDT models are a convenient and simple alternative to WebLogic WLST configuration scripts and templates. They compactly define a WebLogic domain using YAML files and support including application archives in a ZIP file. The WDT model format is described in the open source [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) GitHub project.
+
+For more information on Model in Image, see the [Model in Image user guide]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}), and for a comparison of Model in Image to other domain home source types, see [Choose a domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}).
```
Comment (rosemarymarano) [inline docs-source/content/samples/simple/domains/model-in-image/_index.md]: Two sentences here: 
see the [Model in Image user guide]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}).
For a comparison to other domain home source types, see...

File: docs-source/content/samples/simple/domains/model-in-image/_index.md (hunk 1 above)
Comment (tbarnes-us) [reply docs-source/content/samples/simple/domains/model-in-image/_index.md]: @rosemarymarano Done.  Please resolve the comment and approve the pull you're OK with this.

Comment (rosemarymarano) [reply docs-source/content/samples/simple/domains/model-in-image/_index.md]: ok

File: docs-source/content/samples/simple/domains/model-in-image/_index.md (hunk 2 above)
Comment (rosemarymarano) [reply docs-source/content/samples/simple/domains/model-in-image/_index.md]: @tbarnes-us Is this done yet?

Comment (tbarnes-us) [reply docs-source/content/samples/simple/domains/model-in-image/_index.md]: Fixed. Please resolve this and approve if you think everything is OK now.

**Pull request merged**